src/main/java/DukeException.java

```diff
@@ -0,0 +1,23 @@
+public class DukeException {
+
+    public String message;
+    public boolean isValid;
+
+    public DukeException(String message){
+        this.message = message;
+        this.isValid = true;
+    }
+
+    public DukeException(){
```

Comment:
Is it better to place a white space between the close bracket and the open curly brace?